src/main.rs

```diff
@@ -65,10 +66,12 @@ async fn main() -> anyhow::Result<()> {
                     serde_yaml::to_writer(&file, &kubit::resources::AppInstance::crd())?;
                 }
             }
-            None => println!(
-                "{}",
-                serde_yaml::to_string(&kubit::resources::AppInstance::crd()).unwrap(),
-            ),
+            None => {
+                for crd in crds {
+                    // The YAML delimiter is added in the event we have multiple documents.
+                    println!("---{}", serde_yaml::to_string(&crd).unwrap());
+                }
+            }
```

Comment:
@mkmik Since this code path if dependent on the `crd_dir` flag being present or not, I'm not too sure how we decouple this with a `BufWriter` :thinking: As this seems to satisfy our needs without wrapping some form of an `out_writer` between the `File` and `stdout` conventions.